README.md

```diff
@@ -5,12 +5,11 @@ microservices with ease.
 
 ## Introduction
 
-This chart bootstraps Traefik version 2 as a Kubernetes ingress controller,
-using Custom Resources `IngressRoute`: <https://docs.traefik.io/providers/kubernetes-crd/>.
+Starting with v28.x, this chart now bootstraps Traefik Proxy version 3 as a Kubernetes ingress controller,
+using Custom Resources `IngressRoute`: <https://doc.traefik.io/traefik/v3.0/routing/providers/kubernetes-crd/>.
+
+It's possible to use this chart with Traefik Proxy v2 using v27.x
```

Comment:
Do we want to mention any support policy here (or elsewhere)? E.g. "There is no active support for the v2 app, but you are free to send bugfixes specifically for v2 to the `v27` branch.". Or even just mention support for v2 is as-is and will receive no bugfixes, only possibly app version bumps until v2 is officially EOL.

Reply:
Ah yeah that sounds fine!